.github/workflows/e2e-suite.yml

```diff
@@ -82,8 +86,14 @@ jobs:
           pip install certifi -U && \
           pip install .[obj,dev]
 
+      - name: Set OpenAPI Spec URL
+        if: ${{ inputs.openapi_spec_url != '' }}
+        run: |
+          echo "Using OpenAPI Spec URL: ${{ inputs.openapi_spec_url }}"
+          echo "OPENAPI_SPEC_URL=${{ inputs.openapi_spec_url }}" >> $GITHUB_ENV
+
       - name: Install Package
-        run: make install
+        run: make install SPEC="${{ env.OPENAPI_SPEC_URL }}"
```

Comment:
I think it might be cleaner to just input it directly here. `SPEC="${{ inputs.openapi_spec_url }}"`